sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -120,7 +120,9 @@ object JDBCRDD extends Logging {
       case StringStartsWith(attr, value) => s"${attr} LIKE '${value}%'"
       case StringEndsWith(attr, value) => s"${attr} LIKE '%${value}'"
       case StringContains(attr, value) => s"${attr} LIKE '%${value}%'"
-      case In(attr, value) => s"$attr IN (${compileValue(value)})"
+      case In(null, value) => "NULL"
```

Comment:
Dumb question but does this replacement make sense? can NULL be a predicate? Something that might otherwise render as `SELECT * FROM foo WHERE NULL in ()` now becomes `SELECT * FROM foo WHERE NULL`?